src/test/java/domain/bet/BetTest.java

```diff
@@ -0,0 +1,62 @@
+package domain.bet;
+
+import domain.result.Outcome;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class BetTest {
+
+    @DisplayName("최초의 배팅금액이 0이하이면 예외를 던진다.")
+    @ParameterizedTest
+    @ValueSource(ints = {-100, 0, -1})
+    void constructBetTest(final int money) {
+        assertThrows(IllegalArgumentException.class,
+                () -> new Bet(money))
+                .getMessage().equals("[ERROR]: 배팅금액은 0원 초과여야 합니다.");
+    }
```

Comment:
예외메세지를 통한 정확한 비교 💯
다만 메세지 비교가 의도한대로 되고 있지 않은 것 같아요.  
아래와 같이 예외메세지를 바꾸면 테스트는 통과할까요? 실패할까요?

```suggestion
    @DisplayName("최초의 배팅금액이 0이하이면 예외를 던진다.")
    @ParameterizedTest
    @ValueSource(ints = {-100, 0, -1})
    void constructBetTest(final int money) {
        assertThrows(IllegalArgumentException.class,
                () -> new Bet(money))
                .getMessage().equals("[ERROR]: 배팅금액은 0원 미만이어야 합니다.");
    }
```

Reply:
당연히 된다고 생각했는데 수정하겠습니다..!!
원리를 알아보겠습니다